antarest/study/storage/variantstudy/model/command/remove_link.py

```diff
@@ -152,7 +151,7 @@ def _apply(self, study_data: FileStudy, listener: Optional[ICommandListener] = N
 
             self._remove_link_from_scenario_builder(study_data)
 
-        return self._apply_config(study_data.config)[0]
+        return self.remove_from_config(study_data.config)[0]
```

Comment:
It looks like the second part of the tuple is never used, we can remove it.
Same in `_check_link_exists`, it seems